content/test/unit/service/validations/Validations.spec.ts

```diff
@@ -154,14 +140,14 @@ describe('Validations', function () {
         data: 'data'
```

Comment:
Is the `originalVersion` in `originalMetadata` field always present? When a new entity is deployed should have those files or it will be assumed as V3?

Reply:
It is assumed that `originalVersion` is always set in `originalMetadata`